lib/config/environment.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
   allowAnonymous: toBooleanConfig(process.env.CMD_ALLOW_ANONYMOUS),
   allowAnonymousEdits: toBooleanConfig(process.env.CMD_ALLOW_ANONYMOUS_EDITS),
   allowFreeURL: toBooleanConfig(process.env.CMD_ALLOW_FREEURL),
+  requireFreeURLAuthentication: toBooleanConfig(process.env.CMD_REQUIRE_FREEURL_AUTH),
```

Comment:
Either the environment variable should be written out or the JS variable name should be cut. Let's try to keep the variable names in sync.

Reply:
Okay, I have decided for the long version as *auth* could also mean *authorization*.